src/redux/actionTypes.js

```diff
@@ -0,0 +1,3 @@
+export const REMOVE_BOOK = 'REMOVE_BOOK';
+export const ADD_BOOK = 'ADD_BOOK';
+export const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- Please the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:      
-  `const ADD_BOOK = bookstore/books/ADD_BOOK` 
- `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK` 
- `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`

Reply:
Thanks